src/main/java/gov/nasa/pds/tools/util/ContextProductReference.java

```diff
@@ -109,23 +109,30 @@ public String toString() {
   @Override
   public boolean equals(Object o) {
     boolean isEqual = false;
-    ContextProductReference lidvid = (ContextProductReference) o;
-
-    // if(lidvid.getName().equals("N/A"))
-    // System.out.println("obj 1: " + lidvid.getLid() + ", " +
-    // lidvid.getVersion() + ", " + lidvid.getName() + ", " +
-    // lidvid.getType());
-
-    // Compare Lid and/or version only
-    if (this.lid.equalsIgnoreCase(lidvid.getLid())) {
-      if (this.hasVersion) {
-        if (lidvid.hasVersion() && this.version.equals(lidvid.getVersion())) {
+    if (o instanceof ContextProductReference) {
+      ContextProductReference lidvid = (ContextProductReference) o;
+
+      // if(lidvid.getName().equals("N/A"))
```

Comment:
Maybe just delete this? If it's needed, add a comment saying why it's commented-out, to help reduce doubt in future readers of this code.

Reply:
Sure. I love to remove dead code. Done.

Reply:
I think Jordon already did.